tests/integration/hpe_3par_api_container_test.py

```diff
@@ -62,6 +62,7 @@ def setUpClass(cls):
                 assert pl_data['Enabled'] is False
                 while pl_data['Enabled'] is False:
                     c.enable_plugin(HPE3PAR)
+                    sleep(60)
```

Comment:
can we write a definition to check whether backend is initialized or not. This is a nightly run and we are aware of the configuration. We can wait until all backends are initialized instead of making it a hard coded value.

once all the backends are initialized to we can proceed for the operations.

Reply:
Created hpe_backends_initialize() function to check for backend initialization